build.gradle

```diff
@@ -8,6 +8,11 @@ version = "${project.mod_version}+${project.minecraft_base_version}"
 group = project.maven_group
 
 repositories {
+
+    flatDir {
+        dirs 'owo-lib_local_repo'
+    }
```

Comment:
maybe this too? and some other stuff to be able to merge it in upstream

Reply:
I'm not sure, it won't compile without local libs since they aren't available on maven.

Reply:
The way you have it currently it won't compile either, since you didn't check the libs directory into source control. In any case, I'd much prefer this to use the actual maven, if and when I publish the backported owo version

Reply:
Well it did build that way but I agree ot would be a lot better to get the libs from maven.